apps/web/src/modules/job/apply/pages/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { UserType } from '@modela/dtos'
+import {
+  Button,
+  CircularProgress,
+  Divider,
+  Link,
+  Typography,
+} from '@mui/material'
+import withGuard from 'common/hoc/withGuard'
+
+import SelectResume from '../components/SelectResume'
+import useResume from './hooks'
+import { CardContainer, RootContainer } from './styled'
+
+const JobApplyPage = () => {
+  const { jobId, jobTitle, handleSuccess, loading, resumes, Id, setId } =
+    useResume()
+  if (loading) return <CircularProgress />
```

Comment:
My bad I think that this is loading for fetching data, but actually, it is loading for submitting data.
We should move Circular progress to the submit button (see example in `modules/login/index.tsx`)